stories/typography.js

```diff
@@ -21,19 +22,19 @@ storiesOf('Typography', module)
   .addDecorator(checkA11y)
   .addDecorator(styles(baseStyles))
   .add('Headings', () => (
-    <div>
-      <Heading1>Heading 1</Heading1>
-      <Heading2>Heading 2</Heading2>
-      <Heading3>Heading 3</Heading3>
-      <Heading4>Heading 4</Heading4>
-    </div>
+    <Box>
```

Comment:
Not really necessary to use a `Box` here right? It confused me at first, It kinda looked like you need to wrap your components into the `Box` for being able to use the margin and padding properties. So I would just change it to a div to avoid confusion.

Reply:
I've added some padding now, so the Box isn't just rendering a div. I did this because in [https://github.com/teamleadercrm/teamleader-ui/pull/84](this PR) I'm removing the generally set padding on the preview area.